Cargo.toml

```diff
@@ -149,7 +149,7 @@ coprocessor_plugin_api = { path = "components/coprocessor_plugin_api" }
 time = "0.1"
 tipb = { git = "https://github.com/pingcap/tipb.git" }
 tokio = { version = "1.12", features = ["full"] }
-tokio-timer = "0.2"
+tokio-timer = { git = "https://github.com/you06/tokio-timer-0.2.13.git", branch = "master" }
```

Comment:
Should be forked to tikv org and changes need to get two LGTMs.

Reply:
This dependency needs to be removed once the self-maintained time library that you mentioned in the issue is ready, so it's a temporary solution only. Do we need to fork this crate into tikv org?

Reply:
Yes.

Reply:
I don't have the permission of creating a repo in TiKV org. Can you help me create a repo and forward my commits into it?

Reply:
Now it's at https://github.com/tikv/tokio/tree/tokio-timer-hotfix.